CWE/decisions/Romania.txt

```diff
@@ -29,4 +30,195 @@ political_decisions = {
+			NOT = { has_country_flag = romanian_oil_modernization_complete }
+		}
+		
+		allow = {
+			NOT = { has_country_modifier = economic_malaise }
+			war = no
+			money = 100000
+		}
+		
+		effect = {
+			prestige = 100
+			667 = {
+				trade_goods = petroleum
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 			
+			}
+			672 = {
+				trade_goods = petroleum
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 			
+			}
+			665 = {
+				trade_goods = petroleum
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 		
+			}
+			set_country_flag = romanian_oil_modernization_complete
+			treasury = -75000
+		}
+		ai_will_do = {
+			factor = 1
+		}
+	}
+
+
+
+	oil_modernization_hungarian_region = {
+		picture = "rom_oil"
+		potential = {
+			AND = {
+				owns = 2533
+				owns = 647
+			}
+			OR = {
+				has_country_flag = nicolae_ceausescu_flag
+				has_country_flag = rom_manescu
+				has_country_flag = gheorghe_apostol_romania
+				has_country_flag = rom_nicu
+				has_country_flag = rom_zoia
+			}
+			OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 government = socialist_democracy } }
+			has_country_flag = romanian_oil_modernization_start
+			NOT = { has_country_flag = romanian_oil_modernization_complete_hungarian_region }
+		}
+		
+		allow = {
+			NOT = { has_country_modifier = economic_malaise }
+			war = no
+			money = 25000
+		}
+		
+		effect = {
+			prestige = 50
+			2533 = {
+				trade_goods = natural_gas 
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 
+			}
+			647 = {
+				trade_goods = other_minerals
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+			}
+			set_country_flag = romanian_oil_modernization_complete_hungarian_region
+			treasury = -10000
+		}
+		ai_will_do = {
+			factor = 1
+		}
+	}
+
+
+	project_bucharest = {
+		picture = "renovate_capital"
+		potential = {
+			owns = 664
+			capital_scope = { has_province_modifier = new_capital }
+			NOT = { has_country_flag = germania }
+			NOT = { has_country_modifier = germania }
+			tag = ROM
+			OR = {
+				has_country_flag = nicolae_ceausescu_flag
+				has_country_flag = rom_manescu
+				has_country_flag = gheorghe_apostol_romania
+				has_country_flag = rom_nicu
+				has_country_flag = rom_zoia
+			}
+			OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 government = socialist_democracy } }
+			has_country_flag = systematization_ROM
+			NOT = { has_country_flag = germania }
+		}
+
+		allow = {
+			NOT = { has_country_modifier = economic_malaise }
+			war = no
+			money = 5000000
+			is_disarmed = no	
```

Comment:
is_disarmed = no has misleading localisation in game so this is better: NOT = { is_disarmed = yes }